### PR TITLE
Updates formatting of notice to conform to new standard

### PR DESCRIPTION
**Pull request by schalkneethling**
Fixes formatting of notice

### CODE REVIEW

**Comment by schalkneethling** [comment]
@peiying2 r?

File: firefox-cliqz_privacy_notice/en-US.md [hunk 1]
```diff
@@ -36,34 +36,35 @@ Firefox displays content, such as "Snippets" (messages from Mozilla) and Top Sit
...
-  * select "Extensions" on the left menu bar
-  * if installed, Cliqz will appear on the right
-  * selecting "Disable" will turn off the Cliqz feature and stop related data collection
-  * selecting "Remove" will uninstall the Cliqz feature from Firefox
 
+* type "about:addons" into your Firefox URL bar
+* select "Extensions" on the left menu bar
+* if installed, Cliqz will appear on the right
+* selecting "Disable" will turn off the Cliqz feature and stop related data collection
+* selecting "Remove" will uninstall the Cliqz feature from Firefox
```
Comment (peiying2) [inline firefox-cliqz_privacy_notice/en-US.md]: This block of bullets - should the first letter in each bullet start with capital letter? It seems the style has changed compared to the others.  @brsmith-mozilla @eleemoz could you comment?

Comment (peiying2) [inline firefox-cliqz_privacy_notice/en-US.md]: The bullet part is messed up: https://bedrock-demo-privacy.us-west.moz.works/en-US/privacy/firefox-cliqz/

Comment (brsmith-mozilla) [reply firefox-cliqz_privacy_notice/en-US.md]: I have not been involved with developing this notice (Elvin is on PTO this week), but I believe that each line should start with a capital letter.

Comment (schalkneethling) [reply firefox-cliqz_privacy_notice/en-US.md]: Changed.

Comment (schalkneethling) [reply firefox-cliqz_privacy_notice/en-US.md]: Oh, it is actually generating a bullet list but, the CSS turns of the little dots by default. Nothing to change here, the change is on the bedrock side.

**Pull request merged**